TodoList/Sources/TodoList.Contracts/Api/IBootstrapper.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Microsoft.Practices.Unity;
 
-namespace TodoList.Contracts.Interfaces
+namespace TodoList.Contracts.Api
```

Comment:
`Bootstrapper` is in no way connected with (Web) Api while this namespace suggests such meaning.  Move it to a `.Bootstrap` namespace, please. 

The point there is that this interface is used in many (right now two, true) assemblies and they using statements should not express any particular connection to any particular user-facing facility. For example, if there was a WPF application, it would also need to call (all) bootstraps during its startup, yet it would have nothing in common with (web) API…